javascript/javascript1/week3/item-array-removal.js

```diff
@@ -0,0 +1,28 @@
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
```

Comment:
Everything is fine, Nice and clean , also very good that you made a function for this exercise. You could for instance make the function more reusable by having two parameters.